ibmcloud/image/multipart_upload.sh

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
Added preamble, I prefer to keep it separate. Even though it might not be reference elsewhere it is a independant piece of logic and can be tested separately more easily as its own file